bitnami/node/values.yaml

```diff
@@ -70,16 +70,24 @@ volumePermissions:
 ## ref: https://hub.docker.com/r/bitnami/git/tags/
 ##
 git:
-  registry: docker.io
-  repository: bitnami/git
-  tag: 2.29.1-debian-10-r5
-  pullPolicy: IfNotPresent
-  ## Optionally specify an array of imagePullSecrets.
-  ## Secrets must be manually created in the namespace.
-  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## Bitnami git image version
+  ## ref: https://hub.docker.com/r/bitnami/git/tags/
   ##
-  # pullSecrets:
-  #   - myRegistryKeySecretName
+  image:
```

Comment:
This change should be reflected in the models where the secondary images are defined